lib/python/bins.cpp

```diff
@@ -67,18 +65,15 @@ template <class T> void bind_bins(pybind11::module &m) {
                                         // implicit conversions in functor
 }
 
-template <class T> auto bin_begin_end(const Variable &var) {
+template <class T> py::dict bins_constituents(const Variable &var) {
```

Comment:
If this is `const`, doesn't it prevent modification? Not sure if this results in readonly flags getting set.

Reply:
No it does not. The constituents can be modified.